src/dag/chunk.rs

```diff
@@ -1,8 +1,100 @@
-pub type Key = String;
+use super::meta_generated;
+use flatbuffers;
+use flatbuffers::FlatBufferBuilder;
 
 #[allow(dead_code)]
+#[derive(Debug)]
 pub struct Chunk {
-    pub key: Key,
-    buf: Vec<u8>,
-    refs: [Key],
+    hash: String,
+    data: Vec<u8>,
+    meta: Option<(Vec<u8>, usize)>,
+}
+
+#[allow(dead_code)]
```

Comment:
I think you can remove this and the `allow` on line 5 now.

Reply:
done